src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue

```diff
@@ -186,13 +192,35 @@ export default {
 		},
 	},
 
-	mounted() {
+	watch: {
+		isActive(newValue) {
```

Comment:
So everytime you go to this tab it immediately starts a new sync of the participants and starts the 30s interval.
Should be fine enough to do it on mount only?

Reply:
It's needed, but we're missing the part where we stop the interval if you navigate away from the tab, fixing this now

Reply:
Actually, it's in mounted that it's not needed